.github/workflows/js-test.yaml

```diff
@@ -0,0 +1,82 @@
+name: Run JS Test Cases
+on:	
+  push:	
+    branches:	
+      - master	
+  pull_request:
+    branches:  master 
+jobs:	
+  publish-npmjs:	
+    runs-on: ubuntu-20.04	
+    steps:	
+      - uses: actions/checkout@v3	
+      - uses: actions/setup-node@v3	
+        with:	
+         node-version: '16.x'	
+         registry-url: 'https://registry.npmjs.org'	
+      - uses: actions/setup-java@v3	
+        with:	
+          java-version: '11'	
+          distribution: 'adopt'	
+      - name: Build AMS from source
```

Comment:
Building AMS from source is too much work. Why don't you get latest snapshot from website?

Reply:
It could be much more efficient. Because of Usama is off for 2 weeks, I will refactor this pr. @mekya